docs/source/installation.md

```diff
@@ -16,183 +16,182 @@ limitations under the License.
+</Tip>
 
-Setting environment variable `TRANSFORMERS_OFFLINE=1` will tell 🤗 Transformers to use local files only and will not try to look things up.
+# Offline mode
 
-Most likely you may want to couple this with `HF_DATASETS_OFFLINE=1` that performs the same for 🤗 Datasets if you're using the latter.
+🤗 Transformers is able to run in a firewalled or offline environment by only using local files. Set the environment variable `TRANSFORMERS_OFFLINE=1` to enable this behavior.
 
-Here is an example of how this can be used on a filesystem that is shared between a normally networked and a firewalled to the external world instances.
+<Tip>
+Add 🤗 [Datasets](https://huggingface.co/docs/datasets/) to your offline training workflow by setting the environment variable `HF_DATASETS_OFFLINE=1`.
+</Tip>
 
-On the instance with the normal network run your program which will download and cache models (and optionally datasets if you use 🤗 Datasets). For example:
+For example, you would typically run a program on a normal network firewalled to external world instances with the following command:
 
-```
+```bash
 python examples/pytorch/translation/run_translation.py --model_name_or_path t5-small --dataset_name wmt16 --dataset_config ro-en ...
 ```
 
-and then with the same filesystem you can now run the same program on a firewalled instance:
-```
+Run this same program in an offline instance by:
+
+```bash
 HF_DATASETS_OFFLINE=1 TRANSFORMERS_OFFLINE=1 \
 python examples/pytorch/translation/run_translation.py --model_name_or_path t5-small --dataset_name wmt16 --dataset_config ro-en ...
 ```
-and it should succeed without any hanging waiting to timeout.
 
-#### Fetching models and tokenizers to use offline
+This should run the script without hanging or waiting to timeout.
 
-When running a script the first time like mentioned above, the downloaded files will be cached for future reuse. 
-However, it is also possible to download files and point to their local path instead.
+## Fetch models and tokenizers to use offline
 
-Downloading files can be done through the Web Interface by clicking on the "Download" button, but it can also be handled
-programmatically using the `huggingface_hub` library that is a dependency to `transformers`:
+Another option for using 🤗 Transformers offline is to download the files ahead of time, and then point to their local path when you need to use them offline. There are two ways to do this:
 
-- Using `snapshot_download` to download an entire repository
-- Using `hf_hub_download` to download a specific file
+* Download a file through the user interface on the [Model Hub](https://huggingface.co/models) by clicking on the ↓ icon.
 
-See the reference for these methods in the huggingface_hub
-[documentation](https://github.com/huggingface/huggingface_hub/tree/main/src/huggingface_hub).
+    ![download-icon](/imgs/download.png)
```

Comment:
Images should not be put in this folder anymore. See [here](https://github.com/huggingface/transformers/tree/master/docs#adding-an-image)